org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -232,7 +232,8 @@ public override bool isHoverActive
                         }
 #pragma warning restore CS0612
 #pragma warning restore CS0618
-                        else
+                        // Don't attempt palm check if the interactor is not driven by a hand (i.e. non-handed tracked controllers).
```

Comment:
nit: this new if statement can probably be merged with the next one

```cs
else if (handedness != InteractorHandedness.None &&
XRSubsystemHelpers.HandsAggregator?.TryGetPalmFacingAway(handedness.ToXRNode(), out isPalmFacingAway) ?? true)
{
   hoverActive &= isPalmFacingAway;
}
```

Reply:
Actually...this should probably be this...that way is there is no hand aggreator we fallback to base behavior

```cs
else if (handedness != InteractorHandedness.None &&
XRSubsystemHelpers.HandsAggregator != null && 
XRSubsystemHelpers.TryGetPalmFacingAway(handedness.ToXRNode(), out isPalmFacingAway))
{
   hoverActive &= isPalmFacingAway;
}
```

Reply:
I wish I caught this in a previous review

Reply:
The initial if, involving forcedDeprecatedInput and ArticulatedHandController, will also fall into the if block due to the null coalescing operator ?? true if there is no HandsAggregator.  But this is how MRTKRayInteractor behaved prior to the XRI3 changes, so the current is continuing that behavior and not changing it.  Would you want both changed, or leave using null coalescing op?

```
if (forceDeprecatedInput &&
    xrController is ArticulatedHandController handController &&
    (XRSubsystemHelpers.HandsAggregator?.TryGetPalmFacingAway(handController.HandNode, out isPalmFacingAway) ?? true))
{
        hoverActive &= isPalmFacingAway;
}
```

Reply:
hmmm good point.  

Ok disregard my second comment....can you at least fix me nit :)

```cs
else if (handedness != InteractorHandedness.None &&
XRSubsystemHelpers.HandsAggregator?.TryGetPalmFacingAway(handedness.ToXRNode(), out isPalmFacingAway) ?? true)
{
   hoverActive &= isPalmFacingAway;
}
```

Reply:
@AMollis Sounds good, will do, thanks for review.

Reply:
Else if merged